dashboard/views.py

```diff
@@ -178,6 +178,9 @@ def resource_access_within_week(request, course_id=0):
     total_number_student_df = pd.read_sql(total_number_student_sql, conn, params={"course_id": course_id})
     total_number_student = total_number_student_df.iloc[0,0]
     logger.info(f"course_id {course_id} total student={total_number_student}")
+    if total_number_student == 0:
+        logger.info(f"For grade {grade} request there are no students in the percent grade range for course {course_id}")
```

Comment:
How about "There is no student in the percent grade range {grade} for course {course_id}..."?

Reply:
Done